common/source/docs/common-auxiliary-functions.rst

```diff
@@ -621,6 +625,26 @@ Disarms the vehicle if brought low.
 
 Enabling inverted flight only changes how ArduPilot stabilizes the vehicle. It will stabilize it with a roll of 180 degrees from normal whenever inverted flight is enabled in a stabilized mode. Unless the vehicle is capable of inverted flight, do **NOT** use this option.
 
+.. raw:: html
+
+   </td>
+   </tr>
+   <tr>
+   <td><strong>Winch Enable</strong></td>
+   <td>
+
+Enables Winch operation.A low on this channel relaxes the winch.
+
+.. raw:: html
+
+   </td>
+   </tr>
+   <tr>
+   <td><strong>Winch Control</strong></td>
+   <td>
+
+Controls the speed and direction of the winch.Low: unreel, Middle: stop, High: takeup
```

Comment:
Hi, thanks for this.  The direction is the opposite actually.

High will deploy the line, middle will stop the line from being deployed or retracted, low will retract the line.